examples/water_boiler/water_boiler.py

```diff
@@ -44,15 +63,19 @@ def update(self, boiler_power, dt):
         power = pid(water_temp)
         water_temp = boiler.update(power, dt)
 
-        x += [current_time - start_time]
-        y += [water_temp]
+        x += [round(current_time - start_time, 3)]
+        y += [round(water_temp, 2)]
         setpoint += [pid.setpoint]
 
         if current_time - start_time > 1:
             pid.setpoint = 100
 
         last_time = current_time
+        time.sleep(0.1)
```

Comment:
Is this sleep really necessary? Did it affect you results somehow?

Reply:
sample time is an important impact factor in reality. i think it should remind users.

---
**sleep 0.1s**
```
[20.0, 20.0, 30.94, 41.97, 52.72, 63.72, 74.57, 85.4, 92.34, 95.87, 97.8, 98.85, 99.42, 99.72, 99.88, 
99.97, 100.02, 100.04, 100.06, 100.06, 100.07, 100.07, 100.07, 100.07, ]
```

**sleep 0.2s**
```
[20.0, 20.0, 40.38, 60.82, 82.26, 98.25, 98.51, 100.08, 100.07, 100.08, 100.08, 100.08]
```

Reply:
sorry,  i just review code, sample_time is already a factor.

in example it should set  sample_time  explicitly

a sample_time sleep is a good idea.